demo/kaggle-otto/README.MD

```diff
@@ -16,7 +16,7 @@ The parameter `nthread` controls the number of cores to run on, please set it to
 To install the R-package of xgboost, please run
 
 ```r
-devtools::install_github('tqchen/xgboost',subdir='R-package')
+install.packages("xgboost", repos = "https://cran.r-project.org")
```

Comment:
This `install_github()` call is broken. As described here:

https://github.com/dmlc/xgboost/blame/3e26107a9c54230cfec1981a06996a602f526023/doc/build.rst#L316